rocky/reports/templates/partials/export_report_settings.html

```diff
@@ -40,8 +50,7 @@ <h3>{% translate "Recurrence" %}</h3>
                     <button type="submit" form="generate_report">
                         {% translate "Generate report" %}<span class="icon ti-chevron-right" aria-hidden="true"></span>
                     </button>
-                {% endif %}
-                {% if is_scheduled_report %}
+                {% elif is_scheduled_report %}
```

Comment:
Would this mean the var `show_listed_report_names` is always the inverse of this, and as such can be removed?